tests/locationConfigTests.json

```diff
@@ -66,6 +66,16 @@
             "credentialsProfile": "default"
         }
     },
+    "aws-test-mismatch": {
```

Comment:
There are already two regions with the same name in the config, plz remove this and one of them :)

Reply:
Thanks, I removed it.